flocker/testtools/_base.py

```diff
@@ -104,6 +111,153 @@ def mktemp(self):
         return make_temporary_directory(self).child('temp').path
 
 
+class _SplitEliotLogs(Fixture):
+    """
+    Split the Eliot logs out of Twisted logs.
+
+    Assumes that Twisted logs contain Eliot logs as per
+    ``flocker._redirect_eliot_logs_for_trial``, and that these logs have been
+    attached to a test case as a detail named with the value of
+    ``CaptureTwistedLogs.LOG_DETAIL_NAME``.
+
+    Takes the Eliot logs that are in the Trial logs and splits them into a
+    separate detail that contains only the pretty printed Eliot logs.
+    """
+
+    _ELIOT_LOG_DETAIL_NAME = 'twisted-eliot-log'
```

Comment:
The provenance. The attached eliot logs come from the Twisted logging system, as opposed to somewhere else. 

Merging as-is, since it's easy to change later.